sections/common.md

```diff
@@ -194,6 +194,22 @@ The parsing type can be set on the service level and it is recommended to only s
 
 See the chapter on <a href="https://github.com/Intelecom/sms/blob/master/sections/about.md#message-content">message content</a> for further details about the different parsing types.
 </td></tr>
+<tr><td>strex_verification_timeout</td><td>String</td><td>Used for Strex verification process. Specified time to wait for a end user Strex verification before timing out. </br></br> Valid Values: 0-30</td></tr> 
```

Comment:
Should be Integer?

Reply:
Talking to @ahaeber - he said that this is a String in the API so you need to send it like this:
  {
            "key": "strex_verification_timeout",
            "value": "3"
          },

.. but the field itself will contain Integer values.. 

My thinking was that we should go for the field type, not the content to avoid client errors.